### PR TITLE
Remove Firefox-specific Adblocker detection

### PR DESCRIPTION
**Pull request by jbreckmckye**
Could detecting an adblocker on a user's computer breach EU data privacy laws? [This campaigner](https://twitter.com/alexanderhanff/status/722506381451010048) says so, and he thinks he has the case to prove it. The Guardian is seeking advice on the matter, but for now, we're taking the precaution of 'genericizing' our adblocking - making sure that it can't spot specific extensions on specific browsers - so that all that remains is a heuristic that an adblocker _might_ be installed because an ad _appears_ to have been hidden.

This pull request removes the Firefox AdblockPlus detection path from the inlined detection script.

@sndrs @Calanthe

### CODE REVIEW

**Comment by Calanthe** [comment]
👍  This is interesting! I wonder what will be the advice from the lawyers.

File: common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js [hunk 1]
```diff
@@ -16,17 +16,14 @@ try {
...
-                window.guardian.adBlockers.generic = adStyles.getPropertyValue('display') === 'none';
-
-                // Only tells us if FF ABP is installed - not whether it is active
-                var adMozBinding = adStyles.getPropertyValue('-moz-binding');
-                window.guardian.adBlockers.ffAdblockPlus = !!adMozBinding && adMozBinding.match('elemhidehit') !== null;
 
+                adBlockers.hiddenAds = adStyles.getPropertyValue('display') === 'none';
                 try {
-                    window.guardian.adBlockers.onDetect(window.guardian.adBlockers);
+                    adBlockers.onDetect(adBlockers.hiddenAds);
```
Comment (sndrs) [inline common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js]: maybe `active`, or just `guardian.adsBlocked = true`? at first, i thought this was a collection of the ads which were hidden

Comment (jbreckmckye) [inline common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js]: The object also carries the onDetect method.

Comment (sndrs) [inline common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js]: good point. `guardian.adBlockers.active` then? not a blocker, i just didn't understand what it was at first, and therefore seems possible someone else won't.

Comment (jbreckmckye) [inline common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js]: No, fair enough

**Comment by uplne** [comment]
So in theory it is also illegal to detect if you are using IE9 and thus give you different CSS styles?

**Comment by rich-nguyen** [comment]
@uplne exactly, this line of thinking just doesn't make sense. If we include the browser's capabilities in an interpretation of 'user information', then all responsive websites are breaching the law. Even implicit feature detection would be outlawed.

**Pull request merged**